src/miragevpn_server.md

```diff
@@ -0,0 +1,125 @@
+The MirageVPN server is a unikernel which listens on a port for client
+connections, and provides Internet connectivity (with NAT) for all
+authenticated clients. A single network interface is used for both client
+connections and providing connectivity.
+
+The unikernel will encrypt all packets received from the Internet and send
+them to the respective client. All packets received from a client will be
+decrypted and forwarded to the Internet. If "client-to-client" is enabled
+in the configuration, packets from one client which destination is another
+client will be forwarded by the server.
+
+# Scope of the server
+
+The scope is at the moment limited to IPv4 (no IPv6), the server only listens
+on TCP (no UDP). Only layer 3 networking is supported (tun device), there's no
+support for tap devices.
+
+The server will route all traffic to its default gateway. If "client-to-client"
+is specified, packets for other clients will be forwarded to the specific
+client.
+
+Only a single network interface is used by the server, where both TCP listening
+and forwarding packets (using NAT) is done. NAT will always be used.
+
+## Authentication
+
+The server as is only authenticates via X.509 certificates. It is possible to
```

Comment:
I think we don't enforce this. So presumably you could use a configuration that doesn't check certificates?! I think we "only" warn in that case.

Reply:
see https://github.com/robur-coop/miragevpn/pull/265